sig-scheduling/annual-report-2021.md

```diff
@@ -4,15 +4,16 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
-   -
-   -
-   -
+   - Efficient re-queueing of pods, significantly cutting the number of failed scheduling cycles
+   - Improvements to preemption performance
+   - Simplified plugin configuration in component config
+   - Scheduler simulator: https://github.com/kubernetes-sigs/kube-scheduler-simulator
 
 2. What initiatives are you working on that aren't being tracked in KEPs?
 
-   -
-   -
-   -
```

Comment:
BTW, something like this https://github.com/kubernetes/kubernetes/pull/101946 is not trivial, it's not a refactoring nor perf related, but an api-change and impact the final score for each node, we actually change the formula used in that plugin.

It's following a light-weighted KEP wrote here  https://docs.google.com/document/d/1MT0KVnyeAc4QcKqUsmk-4OvmBhOy6PvBQOGnjugrJ9k.

I am wondering whether any api related change should go with formal KEP process.

Reply:
Wasn't the API change part of the component config v1beta2 graduation?

in any case, I added a line related to enhancements to node resource scoring plugins which https://github.com/kubernetes/kubernetes/pull/101822 is also part of.